testing/simple/show_test.go

```diff
@@ -127,7 +127,7 @@ func TestShowPvc(t *testing.T) {
 		actual := string(output)
 
 		t.Logf("actual %s- ", actual)
-		found := strings.Contains(actual, "pgdata")
+		found := strings.Contains(actual, TestClusterName)
 		if !found {
 			t.Error("could not find pgdata in output")
```

Comment:
This message should change, too. Perhaps `could not find cluster name in output`.